### DISCUSSION TITLE
Upgrading from Appium 2.0 beta to 2.0

### DISCUSSION THREAD

**Discussion by Stanzarectah** [appium/appium#18996] [General]
Hi,

I am currently using  Appium v2.0.0-beta.55 and seeing that now V2.0 has been released what is the best way to upgrade?

Will it break anything?

**Comment by KazuCocoa** [comment]
"Appium" is basically a front end server. So the behavior basically depends on each driver itself. Upgrading the appium version may not cause an issue but you need to check each driver version you currently use

**Comment by jlipps** [comment]
Yes, you should upgrade. `npm uninstall -g appium && npm install -g appium` should do the trick.

**Reply by Stanzarectah** [reply]
Thanks guys will check it out
